docs/apiary.apib

```diff
@@ -112,21 +114,33 @@ Register a new template
 
 
 ### Get the current list of templates [GET /template{?page_size,page_num,attr_format,attr,attr_type,label,sortBy}]
+
 Get the full list of templates with all their associated attributes.
 
 + Parameters
     + page_size: 20 (integer, optional)
     + page_num: 1 (integer, optional)
     + attr_format: both (string, optional)
 
-        Must be one of "both", "single" (if 'data_attrs' and 'config_attrs' are to be omited)
-        or "split" (if 'attrs' list is to be omited). 'attrs' field will always contain all
-        template attributes that are listed by both 'data_attrs' and 'config_attrs'.
+        Must be one of `both`, `single` (if `data_attrs` and `config_attrs` are to be omitted)
+        or `split` (if `attrs` list is to be omitted). `attrs` field will always contain all
```

Comment:
In my opinion if `both` been the last option the text would be more clear, for example:
"Must be one of: single (if `data_attrs` and `config_attrs` are to be omitted), `split` (if `attrs` list is to be omitted) or `both`. `attrs` field will always contain all ...". But it is just a suggestion.